packages/paste-core/components/menu/stories/index.stories.tsx

```diff
@@ -271,6 +281,8 @@ export const MenuDropdown = (): React.ReactNode => {
     registerItem: () => {},
     unregisterItem: () => {},
     tabIndex: -1,
+    unstable_values: {},
```

Comment:
question: why add these?

Reply:
Typescript made me do it for checkbox and radio menu items